app/src/main/contrib/stores/ResponsibilityStore.ts

```diff
@@ -42,7 +42,7 @@ export interface ResponsibilityState extends RemoteContent, HasFormatState {
 interface ResponsibilityStoreInterface extends AltJS.AltStore<ResponsibilityState> {
     fetchResponsibilities(): Promise<Responsibilities>;
 
-    fetchTouchstones(): Promise<Touchstone[]>;
+    fetchTouchstones(groupId?: string): Promise<Touchstone[]>;
```

Comment:
I don't think this should have changed - it refers to the method on the `TouchstoneSource`, which still takes no parameters.